frontend/src/components/samples/SampleTableColumns.tsx

```diff
@@ -103,6 +105,55 @@ export const SAMPLE_COLUMN_DEFINITIONS: { [key in SampleColumnID]: SampleColumn
 		},
 	},
 
+	[SampleColumnID.PARENT_CONTAINER]: {
+		columnID: SampleColumnID.PARENT_CONTAINER,
+		title: 'Parent Container',
```

Comment:
Parent Container is not precise enough (could be name or barcode). We should use "Parent Container Barcode" even if it is too long to our taste.